internal/schema-registry/command_dek_describe.go

```diff
@@ -0,0 +1,75 @@
+package schemaregistry
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+func (c *command) newDekDescribeCommand(cfg *config.Config) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Describe a Data Encryption Key (DEK).",
+		Args:  cobra.NoArgs,
+		RunE:  c.dekDescribe,
+	}
+
+	cmd.Flags().String("name", "", "Name of the Key Encryption Key (KEK).")
```

Comment:
This is unintuitive to me. If I saw `confluent schema-registry dek describe --name xyz`, I would think that "xyz" was the name of the DEK, not the KEK.
```suggestion
	cmd.Flags().String("kek-name", "", "Name of the Key Encryption Key (KEK).")
```